Microsoft.Toolkit.Uwp.UI.Controls/Graph/ProfileCard/ProfileCard.xaml

```diff
@@ -0,0 +1,111 @@
+﻿<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls.Graph">
+
+    <Style x:Key="ProfileCardImageBorderStyle"
+           TargetType="Border">
+        <Setter Property="Width" Value="80" />
+        <Setter Property="Height" Value="80" />
+    </Style>
+
+    <Style x:Key="ProfileCardImageEllipseStyle"
+           TargetType="Ellipse">
+        <Setter Property="Width" Value="64" />
+        <Setter Property="Height" Value="64" />
+        <Setter Property="VerticalAlignment" Value="Center" />
+    </Style>
+
+    <Style x:Key="ProfileCardLargeProfileStackPanelStyle"
+           TargetType="StackPanel">
+        <Setter Property="Height" Value="80" />
+        <Setter Property="Orientation" Value="Horizontal" />
+    </Style>
+
+    <Style x:Key="ProfileCardLargeProfileTextBlockStyle"
+           TargetType="TextBlock">
+        <Setter Property="FontSize" Value="24" />
+        <Setter Property="Margin" Value="0 0 0 4" />
+    </Style>
+
+    <Style TargetType="controls:ProfileCard">
+        <Setter Property="IsTabStop" Value="False" />
+        <Setter Property="Background" Value="{ThemeResource ApplicationPageBackgroundThemeBrush}" />
+        <Setter Property="Foreground" Value="{ThemeResource ApplicationForegroundThemeBrush}" />
+        <Setter Property="FontFamily" Value="{ThemeResource ContentControlThemeFontFamily}" />
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="controls:ProfileCard">
+                    <Grid>
+                        <Grid HorizontalAlignment="Left">
+                            <Grid.Visibility>
```

Comment:
This seems pretty obtuse compared to `Visibility="{Binding DisplayMode, RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource DisplayModeConverter}, ConverterParameter='PictureOnly'}"`, usually TemplateBinding helps in your other cases, but here it won't work as you need the converter.  That way you also share the converter instance.

Reply:
@michael-hawker Done